frontend/src/services/auth.service.ts

```diff
@@ -26,17 +41,72 @@ export const saveTokens = (tokens: AuthTokens, userIdentifier: string) => {
 export const googleLogin = async (
   credentialResponse: CredentialResponse
 ): Promise<User> => {
-  console.log('credentialResponse', credentialResponse);
+  console.log("credentialResponse", credentialResponse);
   try {
     const response = await axios.post<User>(
       AUTH_GOOGLE_LOGIN_URL,
       credentialResponse.credential
     );
-    console.log('google credentials', credentialResponse.credential);
+    console.log("google credentials", credentialResponse.credential);
     return response.data;
   } catch (err: unknown) {
     const error = err as AxiosError;
-    console.log('error response', error.message);
+    console.log("error response", error.message);
     throw err;
   }
 };
+
+// Convert fetchOnlineUsers to TypeScript
+export const fetchOnlineUsers = async (
+  userId: string
+): Promise<OnlineUsersResponse | null> => {
+  try {
+    const response = await fetch("http://localhost:3000/auth/online-users", {
```

Comment:
you should establish the URL as a constant in `../utils/constants.ts` file.
and use that like 
```
const response = await axios.post<User>(
      AUTH_ONLINE_USERS_URL,
    );
```

Reply:
Also change from localhost:3000 to BACKEND_URL